### PR TITLE
fix(DB/SAI): restore Drakuru’s (id: 28802) health to 100% upon charming

### PR DESCRIPTION
**Pull request by vrachv**
Previously, when Drakuru was charmed, its current health remained the same as it was before charming, and only its max health was increased.

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. .q a 12686
2. .go c id 28802
3. use `Scepter of Empowerment`

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1735053928227867164.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO `smart_scripts`
```
Comment (Grimdhex) [inline data/sql/updates/pending_db_world/rev_1735053928227867164.sql:1]: Delete before Insert to avoid any potential conflict

**Review by Grimdhex — DISMISSED** [review]

File: data/sql/updates/pending_db_world/rev_1735053928227867164.sql (hunk 1 above)
Comment (vrachv) [reply data/sql/updates/pending_db_world/rev_1735053928227867164.sql:3]: Fixed. I also added all PK for UPDATE.

File: data/sql/updates/pending_db_world/rev_1735053928227867164.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 28802 AND `id` = 4;
```
Comment (sudlud) [inline data/sql/updates/pending_db_world/rev_1735053928227867164.sql:1]: and source_type = 0

Comment (vrachv) [reply data/sql/updates/pending_db_world/rev_1735053928227867164.sql]: Done

**Review by sudlud — APPROVED** [review]

**Pull request merged**